data.tf

```diff
@@ -0,0 +1,15 @@
+data "aws_iam_policy_document" "route53_query_logs" {
+  statement {
+    actions = [
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    resources = ["arn:aws:logs:*:*:log-group:/aws/route53/*"]
```

Comment:
Please change the resource to point _directly_ to the log-group that you created. This policy is too broad...